pkg/predicate/predicate.go

```diff
@@ -205,6 +205,19 @@ func AddTypePredicate(extensionType string, predicates []predicate.Predicate) []
 	return append(preds, predicates...)
 }
 
+// AddTypesPredicate returns a new slice which contains a types predicate and the given `predicates`.
+// all given types are or combined
+func AddTypesPredicate(extensionTypes []string, predicates []predicate.Predicate) []predicate.Predicate {
```

Comment:
Does it make sense to re-use the `AddTypePredicate` function instead and change its signature from

`func AddTypePredicate(extensionType string, predicates []predicate.Predicate) []predicate.Predicate`

to

`func AddTypePredicate(predicates []predicate.Predicate, extensionType ...string) []predicate.Predicate`

?

Reply:
sounds legit, gimme a second